SPECS/kernel.spec

```
@@ -996,12 +997,13 @@ fi
 %{?_cov_results_package}
 
 %changelog
+* Thu Jun 06 2024 Petr Bena <benapetr@gmail.com> - 4.19.19-8.0.33.2
+- Integrated ceph code from kernel 4.19.295
```

```suggestion
* Thu Jun 06 2024 Petr Bena <benapetr@gmail.com> - 4.19.19-8.0.33.2
- Integrated ceph code from kernel 4.19.295

```

Changelog entries are separated by a newline.